src/app/pages/news/9.3/abstract.html

```diff
@@ -1,10 +1,12 @@
 <div class="row">
 	<div class="4u 12u$(xsmall)">
-		<img class="image fit" src="/images/news/9.2/market-connectors/01-market-browse.png" alt="Release 9.3" />
+		<img class="image fit" src="/images/news/9.3/scaling/scaling-haproxy.png" alt="Release 9.3" />
 	</div>
 	<div class="4u 12u$(xsmall)">
-		<h5>CLUSTER</h5>
-		<p>deployment
+		<h5>Scaling</h5>
+		<p>It was never easier to scale Axon Ivy Engine. 
+       Use the container technology of your choice to scale Axon Ivy Engine vertically to match the current 
+       load of your clients.
```

Comment:
I think we should use more busswords to catch non technical users to: 
- 'cloud' , 'start more engines on demand on heavy load' or similar wordings :)